### PR TITLE
Polish Tenants page

### LINKED ISSUE #4155: Make Tenants page pretty
I notice that Tenants pages has a lot of style issues, for consistency I will make it similar to the content items page

### PR DESCRIPTION
**Pull request by hishamco**
Fixes #4155

### CODE REVIEW

File: src/OrchardCore.Modules/OrchardCore.Tenants/Views/Admin/Index.cshtml [hunk 1]
```diff
@@ -37,52 +37,77 @@
...
+                            <select name="bulkAction" class="form-control">
+                                <option value="@BulkAction.Disable.ToString()">@T["Disable"]</option>
+                                <option value="@BulkAction.Enable.ToString()">@T["Enable"]</option>
+                            </select>
+                        </div>
+                    </div>
+                    <div class="form-group col-xl-9 mb-xl-n1">
+                        <div class="input-group input-group-sm">
+                            <button type="submit" class="btn btn-sm btn-secondary">@T["Apply"]</button>
+                            <a asp-route-action="Create" class="btn btn-sm btn-secondary" role="button" style="position:absolute;right:0">@T["Add Tenant"]</a>
```
Comment (hishamco) [inline src/OrchardCore.Modules/OrchardCore.Tenants/Views/Admin/Index.cshtml]: Replace inline style with class or move it to external style

**Comment by hishamco** [comment]
This is how it looks like:

Tenants

**Comment by Skrypt** [comment]
One small adjustment I'd do. I would rename the action button to Bulk Action and make it a button dropdown. https://getbootstrap.com/docs/4.0/components/dropdowns/#single-button-dropdowns

**Comment by hishamco** [comment]
I agree regarding Bulk Action, but moving them before the filter is inconsistent, have a look to the content items page

**Comment by hishamco** [comment]
Tenants

Hope this is fine

**Comment by Skrypt** [comment]
Works for me. Only missing the select all. We could apply the same kind of change on the Content Items list.

**Comment by sebastienros** [comment]
Remove the colon `:` on the `Bulk Actions` button.
Make the `Add Tenant` green or blue (primary)

**Comment by Skrypt** [comment]
See how Github does it.

image

**Comment by sebastienros** [comment]
Let's not try to add too many things in this PR, or it will diverge and will never get merged.

**Comment by Skrypt** [comment]
Yeah don't do the Content List part in this PR. Make it look better on the Tenants list view if you want to. Or leave it as is for now and we will find a better solution for the select all checkbox later on. I'd say apply the changes that @sebastienros asked for and I approve. Nice work. Thank you @hishamco 😄

**Comment by hishamco** [comment]
Ok, just I will finalize some changes and push it with the modification that we agreed on

**Comment by hishamco** [comment]
Done!!

**Pull request merged**